spec/active_force/active_query_spec.rb

```diff
@@ -1,12 +1,17 @@
 require 'spec_helper'
 
+class TestSObject < ActiveForce::SObject
+  def self.decorate(records)
```

Comment:
There was a spec failing below when I changed `sobject` to a `class_double` because `ActiveForce::SObject` does not implement `.decorate`.  I see that this method is supposed to be something that the consumer would implement in their subclass.  This seems like a weird pattern to use with class methods.  In any case, this seemed like the least intrusive workaround so that we can get a `class_double` here that validates that this mock matches real methods.

Reply:
Yeah, I never fully understood why the original authors of this library did something like that but I guess we're stuck with it for now. 🤷